packages/create-pantheon-decoupled-kit/src/templates/partials/nextjs-shared/searchPageCSSModules.ts

```diff
@@ -0,0 +1,44 @@
+export const searchPageCSSModules = () => /* css */ `.altResult {
+	display: flex;
+	margin: var(--auto);
+	margin-top: var(--12);
+	font-size: var(--5);
+	line-height: var(--7);
+	text-align: center;
+	flex-direction: column;
+}
+
+.section {
+	display: flex;
+	margin: var(--auto);
+	margin-top: var(--10);
+	flex-direction: column;
+	max-height: 100vh;
+}
+
+.container {
+	margin: var(--auto);
+	max-width: var(--xs);
+}
+
+.link {
+	text-decoration-line: underline;
+}
+
+.listTitle {
+	font-weight: 700;
+	font-size: var(--7);
+	margin-bottom: var(--4);
+	margin-top: var(--4);
+	justify-items: start;
+}
+
+@media (min-width: 1024px) {
+	.section {
+		min-width: var(--md);
+	}
+	.container {
+		max-width: var(--lg);
```

Comment:
I don't think we should change anything now, but I'm noticing when I search "hello" and get the single "hello world" post back, the width of the container is `sm`, but with a search like "example", since the excerpts are longer it makes the container width `md` or `lg`. So going between the two searches is a bit jarring. I think we should address this in our future re-theme work.